docs/projects/roman-numeral-conversion/2019-10-27-python.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Roman Numeral to Decimal number in Python
+layout: default
+last-modified: 2019-10-27
+featured-image: 
+tags: [python, roman_numeral_to_decimal_number]
+authors:
+  - auroq(code)
+  - rayavarapuvikram1(article)
```

Comment:
For this, you can just put yourself as the primary author. The parentheses will actually break our system.

```suggestion
  - rayavarapuvikram1
```

Don't forget to add yourself to the author file.